R/importNet.MITAB.R

```diff
@@ -0,0 +1,98 @@
+###psuedocode for psi_import###
+
+
+#A dataframe containing information about whether or not a type of interaction requires the addition of a symmetric edge.

```

Comment:
Is this a prerequisite? That information is not available prior to this function.